src/render.js

```diff
@@ -17,17 +21,27 @@ type Options = {
  */
 export default function render(
   component: React.Element<any>,
-  options?: Options
+  { wrapper: WrapperComponent, createNodeMock }: Options = {}
 ) {
-  const renderer = renderWithAct(component, options);
+  const wrapUiIfNeeded = (innerElement: React.Element<any>) =>
```

Comment:
Um, I think I'd prefer creating the component here once and reuse later:
```jsx
const componentToRender = WrapperComponent ? (
  <WrapperComponent>{component}</WrapperComponent>
) : (
  component
);
```

Reply:
I'll test this. Thought it needed a new component...

Reply:
The new component is only going to be created when WrapperComponent is present, so there's not too much overhead :)

Reply:
It's a function so `update` can call it later with a different `innerElement`. I suppose I could pass `WrapperComponent` to `update` and write this wrapping logic again there or move the function out of the `render` scope to share it.

Reply:
Idk what you intended with your comment but I'm not sure how that would work as it needs to use an updated component later. Something reusable needs to be passed to `update`, either a function or the `WrappedComponent`.

Do you like pulling `wrapUiIfNeeded` out and passing around the `WrappedComponent` any better? (I'm not attached to the names, it's just what the other repo had.)

```js
const wrapUiIfNeeded = (innerElement, WrapperComponent) =>
    WrapperComponent ? (
      <WrapperComponent>{innerElement}</WrapperComponent>
    ) : (
      innerElement
    );

export default function render(
  component,
  { wrapper: WrapperComponent, createNodeMock }
) {
  const renderer = renderWithAct(
    wrapUiIfNeeded(component, WrapperComponent),
    createNodeMock ? { createNodeMock } : undefined
  );

  ...

  const update = updateWithAct(renderer, WrapperComponent);

  ...
}

function updateWithAct(
  renderer,
  WrapperComponent,
) {
  return function(component: React.Element<any>) {
    act(() => {
      renderer.update(wrapUiIfNeeded(component, WrapperComponent));
    });
  };
}
```

Reply:
Sure let's leave it then